commit-msg-hook.js

```diff
@@ -1,4 +1,18 @@
 #!/usr/bin/env node
 
+var fs = require('fs')
+var path = require('path')
+
+// git stores last commit message here:
+var fname = 'COMMIT_EDITMSG'
+
+// assume this hook is .git/hooks/commit-msg
+var fmsg = path.join(path.resolve(__dirname, '../'), fname)
+
+if (!fs.existsSync(fmsg)) {
+  // assume this hook is node_modules/commitplease/commit-msg-hook.js
+  fmsg = path.join(path.resolve(__dirname, '../../.git/' + fname))
```

Comment:
Similar to what that husky issue suggested, `var msg = fs.readFileSync('.git/COMMIT_EDITMSG', 'utf8')`.